### PR TITLE
log: add more log output

### PR DESCRIPTION
**Pull request by alpha-baby**
### Issues associated with this PR

#2391 

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/protocol/xprotocol/boltv2/decoder.go [hunk 1]
```diff
@@ -43,6 +44,14 @@ func decodeRequest(ctx context.Context, data types.IoBuffer, oneway bool) (cmd i
 	contentLen := binary.BigEndian.Uint32(bytes[20:24])
 
 	frameLen := RequestHeaderLen + int(classLen) + int(headerLen) + int(contentLen)
+	if frameLen > bolt.BigFrameFlagSize { // 1MB
```
Comment (taoyuanyuan) [inline pkg/protocol/xprotocol/boltv2/decoder.go:47]: typo？  1GB

File: pkg/protocol/xprotocol/boltv2/decoder.go [hunk 2]
```diff
@@ -43,6 +44,14 @@ func decodeRequest(ctx context.Context, data types.IoBuffer, oneway bool) (cmd i
...
+		writeLength := 256
+		if len(bytes) < writeLength {
+			writeLength = len(bytes)
+		}
+		log.Proxy.Errorf(ctx, "[protocol][boltv2] [decodeRequest] maybe receive big request frame > %d, data: %v", bolt.BigFrameFlagSize, bytes[:writeLength])
+		return nil, bolt.ErrBigFrame
```
Comment (taoyuanyuan) [inline pkg/protocol/xprotocol/boltv2/decoder.go:53]: we should not return error.

Comment (taoyuanyuan) [reply pkg/protocol/xprotocol/boltv2/decoder.go:53]: we should log frameLen.

File: pkg/protocol/xprotocol/bolt/types.go [hunk 3]
```diff
@@ -83,6 +83,9 @@ var (
 	// Encode/Decode Exception
 	ErrUnKnownCmdType = errors.New(UnKnownCmdType)
 	ErrUnKnownCmdCode = errors.New(UnKnownCmdCode)
+	ErrBigFrame       = errors.New("bolt big frame")
```
Comment (taoyuanyuan) [inline pkg/protocol/xprotocol/bolt/types.go:86]: it's not an Err.

Comment (alpha-baby) [reply pkg/protocol/xprotocol/bolt/types.go:86]: bolt 协议这种 rpc 协议，一般来说不会发送很大的包，例如：不会大于 8MB 16MB 或者 32MB，如果大于则说明是协议包出现了错误，这时候是不是就应该看作是一个错误了。
如果返回错误，这样就能立即断开该连接了。

Comment (taoyuanyuan) [reply pkg/protocol/xprotocol/bolt/types.go:86]: sofa协议有定义不能传多大的包吗 我理解我传1G的包也不是错误吧

File: pkg/protocol/xprotocol/boltv2/decoder.go [hunk 4]
```diff
@@ -117,6 +126,14 @@ func decodeResponse(ctx context.Context, data types.IoBuffer) (cmd interface{},
 	contentLen := binary.BigEndian.Uint32(bytes[18:22])
 
 	frameLen := ResponseHeaderLen + int(classLen) + int(headerLen) + int(contentLen)
+	if frameLen > bolt.BigFrameFlagSize { // 1MB
```
Comment (spacewander) [inline pkg/protocol/xprotocol/boltv2/decoder.go:129]: Is it possible to exact the repeated branch into a function?

File: pkg/protocol/xprotocol/bolt/types.go (hunk 3 above)
Comment (chuailiwu) [reply pkg/protocol/xprotocol/bolt/types.go:86]: 大包可以打印一些日志方便问题定位，但是不能直接报错，rpc没有这个限制，长度限制是依赖bolt协议的

**Review by spacewander — APPROVED** [review]

**Review by taoyuanyuan — APPROVED** [review]

**Pull request merged**